Magento_Checkout/web/template/minicart/content.html

```diff
@@ -1,37 +1,12 @@
-<!--
-/**
- * Copyright © 2013-2017 Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
--->
-<h3 class="heading">
-        <span class="text"><!-- ko i18n: 'My Cart' --><!-- /ko --></span>
-        <span
-            class="qty empty"
-            data-bind="css: { empty: !!getCartParam('summary_count') == false },
-                       attr: { title: $t('Items in Cart') }">
-            <!-- ko text: getCartParam('summary_count') --><!-- /ko -->
-        </span>
-</h3>
-
-<div class="block-content minicart__content">
-    <button type="button"
-            id="btn-minicart-close"
-            class="action close minicart__btn-close"
-            data-action="close"
-            data-bind="attr: { title: $t('Close') }">
-            <!-- <img data-bind="attr: { 'data-src': require.toUrl('Magento_Checkout/images/close.svg') }"
-                 alt="close button"
-                 class="minicart__btn-close-img lazyload"
-            /> -->
-            <svg class="minicart__btn-close-icon">
-                <use data-bind="attr: { 'xlink:href': require.toUrl('images/icons-sprite.svg#close') }" xlink:href='' ></use>
-            </svg>
-    </button>
-
+<div class="minicart-content__total">
+    <!-- ko ifnot: getCartParam('summary_count') -->
+        <div class="subtitle empty" data-bind="visible: closeSidebar()">
+            <!-- ko i18n: 'You have no items in your shopping cart.' --><!-- /ko -->
+        </div>
+    <!-- /ko -->
     <!-- ko if: getCartParam('summary_count') -->
-        <div class="items-total">
-            <span class="count"><!-- ko text: getCartParam('summary_count') --><!-- /ko --></span>
+        <div class="minicart-content__total-counter items-total">
```

Comment:
check if we can remove `items-total` class

Reply:
Removed